lldb/source/Commands/CommandObjectStats.cpp

```diff
@@ -83,13 +86,15 @@ class CommandObjectStatsDump : public CommandObjectParsed {
 
     void OptionParsingStarting(ExecutionContext *execution_context) override {
       m_all_targets = false;
+      m_summary_only = false;
     }
 
     llvm::ArrayRef<OptionDefinition> GetDefinitions() override {
       return llvm::ArrayRef(g_statistics_dump_options);
     }
 
     bool m_all_targets = false;
+    bool m_summary_only = false;
```

Comment:
Change this to:
```
lldb_private::StatisticsOptions m_options;
```
We should probably also move `m_all_targets` into `lldb_private::StatisticsOptions`.

Reply:
> We should probably also move `m_all_targets` into `lldb_private::StatisticsOptions`

I can do that but notice the `m_all_targets` was not used anywhere outside of the command object. The target is decided with the current execution context.